packages/package.json

```diff
@@ -20,11 +20,10 @@
 		"build": "yarn lerna run build",
 		"build:try": "yarn lerna run --no-bail build",
 		"watch": "lerna run --parallel build:main -- --watch --preserveWatchOutput",
-		"sync-version-and-changelog": "lerna version --exact --force-publish --conventional-commits --no-git-tag-version --no-push --yes $(node -p \"require('../meteor/package.json').version\")",
-		"postsync-version-and-changelog": "yarn stage-versions",
 		"sync-version": "yarn set-version $(node -p \"require('../meteor/package.json').version\")",
```

Comment:
Is this `sync-version` command used anywhere else still?

Reply:
no it's not, I thought I removed it but must have messed up :)